game/app/views/users/scoreboard.html.erb

```diff
@@ -1,30 +1,32 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Scoreboard</title>
-  </head>
-  <body>
-    <h1>Scoreboard</h1>
-    <table>
-      <thead>
-        <tr>
-          <th>Name</th>
-          <th>Score</th>
-          <th>Date and Time</th>
-        </tr>
-      </thead>
-      <tbody>
-        <% @users.each do |user| %>
-          <tr>
-            <td><%= user.name %></td>
-            <td><%= user.highest_score %></td>
-            <td><%= user.datetime %> </td> 
-          </tr>
-        <% end %>
-      </tbody>
-    </table>
+<div class="back-button"> 
+  <%= link_to "Back", "/", class: "decor-button button-shadow" %> 
+</div>
 
-    <br>
-    <%= link_to "Back", "/" %>
-  </body>
-</html>
\ No newline at end of file
+<h1> Scoreboard </h1> 
+
+
+<table class="table">
+  <caption class="table-caption"> Top 10 players with the highest scores: </caption>
+  <thead>
+    <tr>
+    <th scope=""col>Position</th>
```

Comment:
@lizgarciao can you fix that:  `<th scope=""col>Position</th>` ->  `<th scope="col">Position</th>`